sklearn/preprocessing/_discretization.py

```diff
@@ -129,11 +140,22 @@ class KBinsDiscretizer(TransformerMixin, BaseEstimator):
 
     """
 
-    def __init__(self, n_bins=5, *, encode="onehot", strategy="quantile", dtype=None):
+    def __init__(
+        self,
+        n_bins=5,
+        *,
+        encode="onehot",
+        strategy="quantile",
+        dtype=None,
+        random_state=None,
+        subsample="warn",
```

Comment:
We need to keep the order of the docstring (at least when we will start validating with numpydoc)

```suggestion
        subsample="warn",
        random_state=None,
```